src/content/en/updates/2017/06/headless-karma-mocha-chai.md

```diff
@@ -0,0 +1,137 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: "Testing on Headless Chrome with Karma/Mocha/Chai"
+
+{# wf_updated_on: 2017-06-13 #}
+{# wf_published_on: 2017-06-13 #}
+
+{# wf_tags: headless,testing,karma,mocha,chai #}
+{# wf_featured_image: /web/updates/images/generic/headless-chrome.png #}
+{# wf_featured_snippet: How to setup Karma, Mocha, and Chai to run automated tests on Headless Chrome. #}
+
+# Testing on Headless Chrome with Karma/Mocha/Chai {: .page-title }
```

Comment:
We can use my inline feedback widget to get more insight into how much experience our users have with karma/mocha/chai, or just automated testing in general. I'll add a commit to this PR if you're interested